### PR TITLE
[MODULAR] Marked One Changes Part 5: bargain bin edition no. 2

### PR DESCRIPTION
**Pull request by dawsonkeyes**
## About The Pull Request

This PR slightly changes up how the marked one’s charge attack works, while making it less common and hopefully ensuring that it doesn’t create constant superspeed. It also adds semi-rare spoken lines during teleport attacks, and rebalances the final phase in lieu of having random charges.

The sword got a handful of tweaks — it does even less damage to fauna, 60, down from the 120 of old, but that’s still a lot, and I want to keep it in line with the crusher if not just slightly better DPS-wise given that’s flat damage with no hassle.

## How This Contributes To The Skyrat Roleplay Experience

Doesn’t! This guy’s a megafauna, you can’t really roleplay with him. Wish ya could. Miners gave me general feedback on his balancing, and through testing I’m gonna keep making more changes like this.

## Changelog

:cl:
balance: the Marked One has reduced attack cooldowns and rebalanced attack strategies in his final phase
balance: dragonslayer does slightly less fauna damage and the roll does slightly more stamina damage
fix: charge attacks no longer grant John Marked One infinite superspeed
/:cl:

### CODE REVIEW

**Comment by dawsonkeyes** [comment]
> You currently have a negative Fix/Feature pull request delta of -6. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

nuh uh

**Comment by dawsonkeyes** [comment]
nuh uh

File: modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm [hunk 1]
```diff
@@ -370,6 +368,8 @@
 	face_atom(target)
 	minimum_distance = 0
 	charging = TRUE
+	var/pre_charge_delay = move_to_delay
+	move_to_delay = (pre_charge_delay - 0.4)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm]: Make this value into a define, please.
```suggestion
	move_to_delay -= 0.4
```

File: modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm [hunk 2]
```diff
@@ -380,6 +380,8 @@
 	chargetiles = 0
 	playsound(src, 'modular_skyrat/modules/gladiator/Clang_cut.ogg', 75, 0)
 	animate(src, color = initial(color), 0.5 SECONDS)
+	var/post_charge_delay = move_to_delay
+	move_to_delay = (post_charge_delay + 0.4)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm]: Ditto on this one (same define)
```suggestion
	move_to_delay += 0.4
```

File: modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm [hunk 3]
```diff
@@ -402,6 +404,17 @@
...
 			forceMove(targeted)
+			var/static/list/tele_messages = list(
+							"Hi.",
+							"Hello there.",
+							"Hello.",
+							"Hey.",
+							"Yo.",
+							"Boo.",
+							"Sup.",
+						)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm]:
```suggestion
			var/static/list/tele_messages = list(
				"Hi.",
				"Hello there.",
				"Hello.",
				"Hey.",
				"Yo.",
				"Boo.",
				"Sup.",
			)

```

File: modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm [hunk 4]
```diff
@@ -402,6 +404,17 @@
...
+			if(prob(20))
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm]: Make that `20` into a define, please.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

**Review by GoldenAlpharex — APPROVED** [review]
Looks good, thanks!

**Pull request merged**